app/static/css/colors_and_type.css

```
@@ -0,0 +1,230 @@
+  --mkt-kr:         #225F9C;
+  --mkt-us:         #2F7A4F;
+  --mkt-crypto:     #80592F;
+
+  /* Direction (KR convention: gain = red, loss = blue) */
+  --profit-pos:     #9A1C1C;   /* 상승 / 빨강 */
+  --profit-neg:     #0F4C9C;   /* 하락 / 파랑 */
+
+  /* ---- Type ------------------------------------------------------------ */
+  --font-sans: "Pretendard Variable", "Pretendard", "Noto Sans KR",
+               "Apple SD Gothic Neo", system-ui, -apple-system, "Segoe UI",
+               Roboto, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)